frontend-ssr/components/search/Search.tsx

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
```

Comment:
Something is wrong with the search, it doesn't work. It seems to be making 2 requests, and the last one is without `query` param.

Reply:
When you fix it test this scenario with a lower PAGE_SIZE so we have more than 1 page:

1. Go on page
2. Search some mambo-jumbo string that has no results  -> should see 0 results in page and the observer should be disconnected
3. Delete the search string -> first page of results should appear
4. Scroll to the bottom -> second page should be fetched